### DISCUSSION TITLE
Can I use my normal Gtk theme?

### DISCUSSION THREAD

**Discussion by ghost** [bottlesdevs/Bottles#159] [Q&A] [answered]
I don't like the normal bottles theme,I want to use my standard Gtk system theme.

**Comment by mirkobrombin** [comment]
The latest AppImage bundle the GTK dependencies to prevent lack of dependencies. I'm already working on the theme problem.

**Comment by mirkobrombin** [comment, marked as answer]
This is already fixed on latest builds.
